code/src/App.js

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
 import data from './data.json';
+import Header from './components/Header';
+import Albums from './components/Albums';
 
 console.log(data);
```

Comment:
You could delete the console.logs when you're finished but that's just a minor detail and doesn't really bother us.
/Ylva & Emma